consensus/fbft_log.go

```diff
@@ -326,17 +327,23 @@ func (consensus *Consensus) ParseNewViewMessage(msg *msg_pb.Message) (*FBFTMessa
 		utils.Logger().Warn().Err(err).Msg("ParseViewChangeMessage failed to parse senderpubkey")
 		return nil, err
 	}
-	FBFTMsg.SenderPubkey = pubKey
-	copy(FBFTMsg.SenderPubkeyBytes[:], vcMsg.SenderPubkey[:])
 
+	FBFTMsg.SenderPubkey = &shard.BLSPublicKeyWrapper{Object: pubKey}
+	copy(FBFTMsg.SenderPubkey.Bytes[:], vcMsg.SenderPubkey[:])
+
+	members := consensus.Decider.Participants()
```

Comment:
can we also cache participants to mask per epoch? can be done in the next PR.

Reply:
yea, it will be done in next PR.